tests/test_pals_settings.py

```diff
@@ -39,6 +39,15 @@
 default_exe = sys.executable
 default_kwargs = {"fail_if_missing_exec": False}
 
+@pytest.fixture(autouse=True)
+def turnoff_telemetry_inderict(monkeypatch):
```

Comment:
interdict?

Reply:
believe it or not it was supposed to be _indirect_ 🤦 

Spelling is hard :(